src/scenarios/staticconsoletemplate/test.py

```diff
@@ -0,0 +1,18 @@
+'''
+C# Console app
+'''
+import sys
+import os
```

Comment:
these imports are not used. should we delete them?

Reply:
done